tests/IntegrationTest.php

```diff
@@ -25,20 +25,20 @@ public function testServiceWiring(): void
             'yesterday' => (new \DateTime('-1 day')),
         ]);
 
-        $this->assertStringContainsString('Yesterday: 1 day ago', $result);
-        $this->assertStringContainsString('Now: now', $result);
-
-        $this->assertStringContainsString('1 day ago', $result);
-        $this->assertStringContainsString('Zero: < 1 second', $result);
-        $this->assertStringContainsString('Less than a second: < 1 second', $result);
-        $this->assertStringContainsString('One second: 1 second', $result);
-        $this->assertStringContainsString('Multiple seconds: 59 seconds', $result);
-        $this->assertStringContainsString('One minute: 1 minute', $result);
-        $this->assertStringContainsString('Multiple minutes: 59 minutes', $result);
-        $this->assertStringContainsString('One hour: 1 hour', $result);
-        $this->assertStringContainsString('Multiple hours: 23 hours', $result);
-        $this->assertStringContainsString('One day: 1 day', $result);
-        $this->assertStringContainsString('Multiple days: 99 days', $result);
+        $this->assertStringContainsString("Yesterday: 1 day ago\n", $result);
+        $this->assertStringContainsString("Now: now\n", $result);
+
+        $this->assertStringContainsString("1 day ago\n", $result);
+        $this->assertStringContainsString("Zero: < 1 second\n", $result);
+        $this->assertStringContainsString("Less than a second: < 1 second\n", $result);
+        $this->assertStringContainsString("One second: 1 second\n", $result);
+        $this->assertStringContainsString("Multiple seconds: 59 seconds\n", $result);
+        $this->assertStringContainsString("One minute: 1 minute\n", $result);
```

Comment:
What about 61 seconds? Does that still say 1 minute? Same for 1 hour and 1 minute (e.g. 3660 seconds)

Reply:
> Does that still say 1 minute?

Yes, I added some tests for this.